opentimelineio/adapters/cmx_3600.py

```diff
@@ -104,8 +104,15 @@ def add_clip(self, line, comments, rate=24):
         comment_handler = CommentHandler(comments)
         clip_handler = ClipHandler(line, comment_handler.handled, rate=rate)
         clip = clip_handler.clip
+        reel = clip_handler.reel
+        if reel != 'AX':
```

Comment:
Is there any harm in setting `clip.metadata['cmx_3600']['reel']` to `AX` (is this `if`necessary)?

Reply:
I was trying to use a light touch.  Creating the cmx_3600 meta object and setting the AX reel breaks a number of the existing tests.  
I personally don't see any _harm_ in setting it.  But if this is the only piece of cmx metadata and we are using AX as  the equivalent of undefined or unknown I can see the argument to omit adding it

Reply:
Can you add a comment to that effect?  That would help clarify why its being skipped.